src/Time/Units.hs

```diff
@@ -144,18 +142,18 @@ __Usage example:__
 "3h5m3s"
 
 -}
-type family (...) (a :: Rat) (b :: Rat) where
+type family (from :: Rat) ... (to :: Rat) where
     x ... y = TakeWhileNot y (DropWhileNot x AllTimes)
 
 -- Drops wile not the required time unit in 'AllTimes'.
-type family DropWhileNot (a :: Rat) (c :: [Rat]) where
+type family DropWhileNot (from :: Rat) (units :: [Rat]) where
     DropWhileNot x '[] = '[]
-    DropWhileNot x (u ': units) = If (u ==% x) (u ': units) (DropWhileNot x units)
+    DropWhileNot x (u ': units) = If (u == x) (u ': units) (DropWhileNot x units)
 
 -- Takes while not equal to the provided bound.
-type family TakeWhileNot (a :: Rat) (c :: [Rat]) where
+type family TakeWhileNot (to :: Rat) (units :: [Rat]) where
     TakeWhileNot x '[] = '[]
-    TakeWhileNot x (u ': units)  = If (u ==% x) '[u] (u ': TakeWhileNot x units)
+    TakeWhileNot x (u ': units) = If (u == x) '[u] (u ': TakeWhileNot x units)
```

Comment:
This is magic to me...  :sparkles:  I wasn't aware that `==` just works out of the box :ballot_box: